hack/kind-test.sh

```diff
@@ -25,7 +25,7 @@ REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 TAG_NAME=$(date "+gmp-%Y%d%m_%H%M")
 TEST_ARGS=""
 # Convert kind cluster name to required regex if necessary.
-KIND_CLUSTER=`echo ${GO_TEST} | sed -r 's/([A-Z])/-\L\1/g' | sed 's/^-//'`
+KIND_CLUSTER=`echo ${GO_TEST} | sed -r 's/([A-Z])/-\L\1/g' | sed 's/^-//' | head -c32`
```

Comment:
suggestion: tests would probably use similar prefixes for tests so I don't think trimming down to 32 is a foolproof. Maybe we should trim it a little more and add a random number at the end?

Reply:
Sure - lemme change this to point to `main` and just consider addressing the cluster name bug.

Reply:
Ok done - PTAL!